### PR TITLE
Add Traditional Chinese `zh_tw.json`

### PR DESCRIPTION
**Pull request by yichifauzi**
Add Traditional Chinese localization

### CODE REVIEW

**Comment by Celibistrial** [comment]
Looks good.
Could you also add your name to the contributors list at `fabric.mod.json`

File: src/main/resources/assets/freelook/lang/zh_tw.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "freelook.key.category": "Freelook 《自由視角》",
```
Comment (notlin4) [inline src/main/resources/assets/freelook/lang/zh_tw.json:2]:
```suggestion
  "freelook.key.category": "Freelook《自由視角》",
```

File: src/main/resources/assets/freelook/lang/zh_tw.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "freelook.key.activate": "啟動自由視角",
+  "freelook.key.menu": "開啟自由視角選單",
+  "freelook.menu.toggle": "自由視角設定為切換",
+  "freelook.menu.hold": "自由視角設定為按住",
+  "freelook.menu.perspective.first": "自由視角設定為第一人稱",
+  "freelook.menu.perspective.second": "自由視角設定為第二人稱",
+  "freelook.menu.perspective.third": "自由視角設定為第三人稱",
+  "freelook.menu.title": "Freelook 《自由視角》設定"
```
Comment (notlin4) [inline src/main/resources/assets/freelook/lang/zh_tw.json:10]:
```suggestion
  "freelook.menu.title": "Freelook《自由視角》設定"
```

File: src/main/resources/assets/freelook/lang/zh_tw.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "freelook.key.category": "Freelook 《自由視角》",
+  "freelook.key.activate": "啟動自由視角",
+  "freelook.key.menu": "開啟自由視角選單",
+  "freelook.menu.toggle": "自由視角設定為切換",
```
Comment (notlin4) [inline src/main/resources/assets/freelook/lang/zh_tw.json:5]:
```suggestion
  "freelook.menu.toggle": "自由視角設為切換模式",
```

File: src/main/resources/assets/freelook/lang/zh_tw.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  "freelook.menu.hold": "自由視角設定為按住",
```
Comment (notlin4) [inline src/main/resources/assets/freelook/lang/zh_tw.json:6]:
```suggestion
  "freelook.menu.hold": "自由視角設為按住模式",
```

File: src/main/resources/assets/freelook/lang/zh_tw.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  "freelook.menu.perspective.first": "自由視角設定為第一人稱",
```
Comment (notlin4) [inline src/main/resources/assets/freelook/lang/zh_tw.json:7]:
```suggestion
  "freelook.menu.perspective.first": "自由視角設為第一人稱",
```

File: src/main/resources/assets/freelook/lang/zh_tw.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  "freelook.menu.perspective.second": "自由視角設定為第二人稱",
```
Comment (notlin4) [inline src/main/resources/assets/freelook/lang/zh_tw.json:8]:
```suggestion
  "freelook.menu.perspective.second": "自由視角設為第二人稱",
```

File: src/main/resources/assets/freelook/lang/zh_tw.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  "freelook.menu.perspective.third": "自由視角設定為第三人稱",
```
Comment (notlin4) [inline src/main/resources/assets/freelook/lang/zh_tw.json:9]:
```suggestion
  "freelook.menu.perspective.third": "自由視角設為第三人稱",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Review by Celibistrial — CHANGES_REQUESTED** [review]
Looks good.
Add your names to the contributors list at `fabric.mod.json`

File: src/main/resources/fabric.mod.json [hunk 8]
```diff
@@ -13,7 +13,9 @@
     "fantahund",
     "TimDev9492",
     "Siratox",
-    "Seeloewen"
+    "Seeloewen",
+    "dirtTW",
+    "Notlin4"
```
Comment (notlin4) [inline src/main/resources/fabric.mod.json:18]:
```suggestion
    "notlin4"
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Comment by Celibistrial** [comment]
Thank you

**Pull request merged**